runspec.sh

```diff
@@ -23,6 +23,10 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 set -euo pipefail
 
-dotnet build --configuration Release src/Feersum/Feersum.fsproj > /dev/null && \
-    dotnet src/Feersum/bin/Release/net5.0/Feersum.dll ${specname} --outputtype exe -o ${specbin} && \
+# FIXME: we shouldn't need to reference Feersum.Core here. The compiler needs
+#        updating to include it implicitly just like `Serehfa`.
+dotnet build --configuration Release Feersum.sln > /dev/null && \
+    dotnet src/Feersum/bin/Release/net5.0/Feersum.dll ${specname} \
+        --reference src/Feersum.Core/bin/Release/net5.0/Feersum.Core.dll \
```

Comment:
This should be added to the base compiler references. Might want another flag in there to _disable_ implicit core references too.